lib/rawlink_lowlevel.mli

```diff
@@ -0,0 +1,13 @@
+type driver = AF_PACKET | BPF
```

Comment:
I tried to not expose `driver` in the mli, but then the compiler warns that the constructors are never used to build values (warn 37). They are built in the C stubs. I guess we can unexpose it and silence the warning in the .ml file.